Automattic/jetpack
# Shortcode Embeds: Fix compat Lazy Loading and YouTube start times

Fixes p1HpG7-ccV-p2

#### Changes proposed in this Pull Request:

* Uses double-quotes in the iframe for YouTube and Archive.org shortcodes to enable support for Lazy Loading.
* YouTube: Add additional support for more start time URL styles.

#### Jetpack product discussion


* p1HpG7-ccV-p2 re lazy loading
* p1623433589233700-slack-C029LRAU2 re YouTube

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:
* To test quotes: Add some YouTube embeds into a post, then run jQuery('iframe').each((k, el) => console.log(el.loading)) from the console.

It should report something like (10) lazy where 10 is the number of embeds.
* To test the YouTube change:
* Connected Jetpack, enable the Shortcodes module so Jetpack takes over YouTube oembed from Core.
* Insert https://youtu.be/o-IvKy3322k?t=10683 to a post. Before this patch, it'll start at 0:00. After, it'll start at the proper time.
